internal/immich/immich.go

```diff
@@ -241,3 +243,8 @@ func FlushApiCache() {
 func ApiCacheCount() int {
 	return apiCache.ItemCount()
 }
+
+func GetApiCacheKey(apiUrl string, requestConfig *config.Config) string {
```

Comment:
I have tried to omit "Get" from the front of vars/funcs. As I feel it's implied in most cases e.g. `i.RandomImage` instead of `i.GetRandomImage`